### PR TITLE
Nordic Way 3 TTL and qpid memory tuning from nw2

### PR DESCRIPTION
**Pull request by RogerMartinsen**
Message time to live (ttl) is respected when forwarding a message in interchange app and message collector.

Helm charts for QPID memory tuning is taken from the work done on Nordic Way 2.

### CODE REVIEW

File: federation-st/src/test/java/no/vegvesen/ixn/federation/FederationSystemST.java [hunk 1]
```diff
@@ -12,40 +12,134 @@
...
+		}
+		reconnectsProducer++;
+	}
+
+	public String randomString(int length) {
+		int leftLimit = 48; // numeral '0'
+		int rightLimit = 122; // letter 'z'
+		Random random = new Random();
+		return random.ints(leftLimit, rightLimit + 1)
+				.filter(i -> (i <= 57 || i >= 65) && (i <= 90 || i >= 97))
```
Comment (kyrrek) [inline federation-st/src/test/java/no/vegvesen/ixn/federation/FederationSystemST.java]: Why are these values as they are? For inctance, why is the lower bound for i 57, when the leftLimit is 48? The same goes for the other side. Also, has to run several random.next()'s which will be filtered out both at both ends and the middle.

File: message-collector/src/test/java/no/vegvesen/ixn/federation/messagecollector/MessageForwarderTest.java [hunk 2]
```diff
@@ -18,18 +15,19 @@
 public class MessageForwarderTest {
```
Comment (kyrrek) [inline message-collector/src/test/java/no/vegvesen/ixn/federation/messagecollector/MessageForwarderTest.java]: Probably best to change to MessageCollector test, and rename some variables

File: message-collector/src/test/java/no/vegvesen/ixn/federation/messagecollector/MessageForwarderTest.java [hunk 3]
```diff
@@ -18,18 +15,19 @@
...
+
+        Source source = mock(Source.class);
+        Sink sink = mock(Sink.class);
 
-        MessageProducer producer = mock(MessageProducer.class);
-        MessageConsumer consumer = mock(MessageConsumer.class);
-        when(collectorCreator.setupCollection(two)).thenReturn(new MessageCollectorListener(consumer,producer));
+        when(collectorCreator.setupCollection(two)).thenReturn(new MessageCollectorListener(sink,source));
 
         MessageCollector forwarder = new MessageCollector(neighbourFetcher, collectorCreator);
```
Comment (kyrrek) [inline message-collector/src/test/java/no/vegvesen/ixn/federation/messagecollector/MessageForwarderTest.java]: Rename to collector

**Review by kyrrek — CHANGES_REQUESTED** [review]
Really good overall, but see the small comments.

File: federation-st/src/test/java/no/vegvesen/ixn/federation/FederationSystemST.java (hunk 1 above)
Comment (RogerMartinsen) [reply federation-st/src/test/java/no/vegvesen/ixn/federation/FederationSystemST.java]: I took inspiration from code that generated readable characters. I don'n need this limitation on printed characters.

**Pull request merged**